src/main/webapp/schema/schema.json

```diff
@@ -244,6 +245,10 @@
               "type": "string",
               "maxLength": 350
             },
+            "supportsMessage": {
```

Comment:
Let's maybe be a bit more specific here and say that it is a 'fileSupportMessage'.

Reply:
Done